roles/haproxy/templates/etc/haproxy/haproxy_swift.cfg

```diff
@@ -23,25 +23,22 @@ defaults
   stats refresh 10s
   stats uri /stats
 
-{% for name, clear_port, enc_port in
-  [
-    [ 'swift', 8080, 8090 ],
-  ]
-%}
-
-frontend {{ name }}
+frontend swift
```

Comment:
I dont see use of it being parameterized.. I believe the template was carried over from haproxy_openstack.cfg.. Moreover, before values were static.. Now all port values are parametrized based on environment